jobs/bump-jenkins-plugins.Jenkinsfile

```diff
@@ -121,11 +120,16 @@ node {
                 withCredentials([usernamePassword(credentialsId: botCreds,
                                                   usernameVariable: 'GHUSER',
                                                   passwordVariable: 'GHTOKEN')]) {
-                                                    shwrap("""
-                                                        cd fedora-coreos-pipeline
-                                                        git push -f https://\${GHUSER}:\${GHTOKEN}@github.com/${fork_repo} main:${pr_branch}
-                                                        curl -H "Authorization: token ${GHTOKEN}" -X POST -d '{ "title": "${message}", "head": "${pr_branch}", "base": "main" }' https://api.github.com/repos/${fork_repo}/pulls
-                                                    """)
+                    // Push to the forked repo
+                    shwrap("""
+                        cd fedora-coreos-pipeline
+                        git push -f https://\${GHUSER}:\${GHTOKEN}@github.com/${fork_repo} main:${pr_branch}
+                    """)
+
+                    // Create a PR against "coreos: fedora-coreos-pipeline:main" repo
+                    def response = shwrapCapture("""
```

Comment:
Do we need to define the variable `response`?

Reply:
may be not. I had added it there initially to capture the output. I can remove it